lib/lotus/assets/helpers/image_tag.rb

```diff
@@ -0,0 +1,15 @@
+module Lotus
+  module Assets
+    module Helpers
+      class ImageTag
+        def self.render(configuration, source, html_options = {})
+          definition = configuration.asset(:image)
+          path       = definition.url(configuration, source)
+          alt        = html_options[:alt] || Lotus::Utils::String.new(File.basename(source, File.extname(source))).titleize
+
```

Comment:
@jodosha 

So I have the html options that the user may has pass, but I have got an static tag definition.
```
tag %(<img src="%s" alt="%s">)
```
If I wanted to add `id` and `class` I could add it to the definition, but they may also be blank; also, the user way want to pass other things beside id or class, so this could could potentially grow too much.

Another option is to gsub our way into the string we want.

Ideally we could use the nice `html` helper that we have in lotus views. Is it possible, or do you have some other clever way to solve this?